components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,160 @@
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="happy"
+                        src={happy}
+                        alt="happy emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["meh"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="meh"
+                        src={meh}
+                        alt="meh emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["crying"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="crying"
+                        src={crying}
+                        alt="crying emoji"
+                        width={width}
+                    />
+                </button>
+            </>
+        );
+    };
+    return (
+        <>
+            <h3 className="mb-4">Send Feedback</h3>
+            {selectedEmoji ? (
+                <>
+                    <div className="flex flex-col -mx-6 px-6 py-4 border-t border-b border-gray-200 dark:border-gray-800 bg-white dark:bg-gray-900">
+                        <div className="relative">
+                            <div className="absolute flex bottom-5 right-5">{emojiGroup(24)}</div>
+                            <textarea
+                                style={{ height: "160px", borderRadius: "6px" }}
+                                autoFocus
+                                className="w-full resize-none text-gray-500 dark:text-gray-400 bg-white"
+                                name="name"
+                                value={text}
+                                placeholder="Have more feedback?"
+                                onChange={(e) => setText(e.target.value)}
+                            />
+                        </div>
+                        <div>
+                            <p className="text-gray-500">
+                                {" "}
+                                By submitting this form you acknowledge that you have read and understood Gitpod’s{" "}
```

Comment:
**question(non-blocking)**: Do you think using Gitpod's policy here is better than our policy as described in https://github.com/gitpod-io/gitpod/pull/10061#discussion_r875784777? Asking as we've been using the second pattern elsewhere in the dashboard. Your call.